src/components/App/ExpressRecorder.tsx

```diff
@@ -244,6 +244,11 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
         this.createStream(this.state.constraints);
     }
 
+    componentDidUpdate() {
+        const button = document.querySelector("button");
+        button && button.focus();
```

Comment:
I would do it in startRecording function.
Also - better to use meaningful class name for the button selection

Reply:
@Channa53  fixed, using ref